src/sentry/testutils/pytest/fixtures.py

```diff
@@ -233,41 +224,27 @@ def default_team(factories, default_organization):
     return team
 
 
-@pytest.mark.django_db
 @pytest.fixture(scope="function")
 def default_project(factories, default_team):
     return factories.create_project(name="Bar", slug="bar", teams=[default_team])
 
 
-@pytest.mark.django_db
 @pytest.fixture(scope="function")
 def default_projectkey(factories, default_project):
     return factories.create_project_key(project=default_project)
 
 
-@pytest.mark.django_db
 @pytest.fixture(scope="function")
 def default_environment(factories, default_project):
     return factories.create_environment(name="development", project=default_project)
 
 
-@pytest.mark.django_db
 @pytest.fixture(scope="function")
 def default_group(factories, default_project):
     # こんにちは konichiwa
     return factories.create_group(project=default_project, message="\u3053\u3093\u306b\u3061\u306f")
 
 
-@pytest.mark.django_db
-@pytest.fixture(scope="function")
-def default_event(factories, default_group):
-    return factories.store_event(
-        data={"event_id": "a" * 32, "message": "\u3053\u3093\u306b\u3061\u306f"},
-        project_id=default_project.id,
```

Comment:
mypy pointed out this is impossible (enabled by some improved typing in `pytest` in 8.x)